### PR TITLE
Feature/toggle notes window location

### PR DESCRIPTION
**Pull request by petermao**
For now, this is just ff33ab6.

New function and keybinding (`M-T`) to toggle between horizontal and vertical doc/note layout.  I know that I'm leaving the separated notes buffer hanging, but I don't use it myself.

Apologies for the serial branching...

### CODE REVIEW

**Comment by petermao** [comment]
This should be much less controversial than PR #26, so I'm going to move the relevant commit parallel to that PR.  I'm not sure if that will junk this PR, though...

File: org-noter-core.el [hunk 1]
```diff
@@ -1845,6 +1845,26 @@ See `org-noter-notes-window-behavior' for more information."
...
+           ((eq current-notes-location 'vertical-split)
+            (setf (org-noter--session-window-location session) 'horizontal-split)))
+     (let (exists)
+       (dolist (window (get-buffer-window-list notes-buffer nil t))
+         (setq exists t)
+         (with-selected-frame (window-frame window)
+           (if (= (count-windows) 1)
+               (delete-frame)
+             (delete-window window))))
+       (when exists (org-noter--get-notes-window 'force))))))
```
Comment (petermao) [inline org-noter-core.el:1866]: @dmitrym0: you could argue that the chunk starting from L1859 should be its own function, since it is lifted straight from `org-noter-set-notes-window-location`.  But without prodding, I take the easy route...

**Comment by dmitrym0** [comment]
> For now, this is just [ff33ab6](https://github.com/org-noter/org-noter/commit/ff33ab629d71606858b8e82d2b74203e81555c59).
> Apologies for the serial branching...

The way branches are created (usually), is from `master`.

So you have

`Master->feature_branch_a->commit1...N`

`Master->feature_branch_b->commit1...N`

This avoid you have to specify which commit you're working on specifically and the diff is easy to read because it contains only the code that belogns to that feature.

I gets a little confusing when branches are split of feature branches (`Master->feature_branch_a->commit1->master_feature_branch_b`) -- makes it a little more difficult to follow what is part of feature_a vs feature_b.

This has been traditionally known as git-flow model (https://nvie.com/posts/a-successful-git-branching-model/).

**Comment by petermao** [comment]
I shamed myself into making the new private function `org-noter--relocate-notes-window`.  I'm not beholden to the name, so if you hate it, let's come up with a better one and then merge this PR.

I'll be better about keeping different features isolated into their own branches going forward.

File: org-noter-core.el [hunk 2]
```diff
@@ -937,6 +937,7 @@ NOTER_PAGE)."
                           (org-noter--session-frame session)))))
 
 (defun org-noter--get-notes-window (&optional type)
+  "Conjure the notes-window from the void."
```
Comment (dmitrym0) [inline org-noter-core.el:940]: 😂 love it.

File: org-noter-core.el [hunk 3]
```diff
@@ -1845,6 +1858,19 @@ See `org-noter-notes-window-behavior' for more information."
               (org-entry-put nil org-noter--property-behavior (format "%s" chosen-behaviors))
             (org-entry-delete nil org-noter--property-behavior))))))))
 
+(defun org-noter-toggle-notes-window-location ()
```
Comment (dmitrym0) [inline org-noter-core.el:1861]: I feel like I'm nitpicking so feel free to ignore -- is it "location" or "orientation"?

**Review by dmitrym0 — APPROVED** [review]

**Comment by dmitrym0** [comment]
I looked through the code it looks good to me, but I haven't ran this locally yet.

What's the best way to test this @petermao? If we had a step by step guide we could roll the instructions into the README as well.

**Comment by petermao** [comment]
README was updated in ff33ab629d71606858b8e82d2b74203e81555c59 as part of this PR.

tldr: new keybinding: `M-T`.

**Comment by petermao** [comment]
I'll wait until you have a chance to try it out before merging this PR.

**Comment by dmitrym0** [comment]
@petermao lovely, works for me!
